src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -52,7 +52,7 @@ class FrameworkExtension extends Extension
      */
     public function load(array $configs, ContainerBuilder $container)
     {
-        $loader = new XmlFileLoader($container, new FileLocator(__DIR__.'/../Resources/config'));
+        $loader = new XmlFileLoader($container, new FileLocator(dirname(__DIR__).'/Resources/config'));
```

Comment:
Just asking: is this kind of change needed to solve the GAE issues?  I agree with @xabbuh that `dirname(__DIR__)` is less readable than `/../` (and it also introduces a function call).